automatic/gimp/gimp.nuspec

```diff
@@ -16,9 +17,8 @@ GIMP has many capabilities. It can be used as a simple paint program, an expert
 
 GIMP is expandable and extensible. It is designed to be augmented with plug-ins and extensions to do just about anything. The advanced scripting interface allows everything from the simplest task to the most complex image manipulation procedures to be easily scripted.
     </description>
-    <summary>GIMP</summary>
-    <releaseNotes></releaseNotes>
-    <copyright>GPL</copyright>
+    <releaseNotes>http://www.gimp.org/release-notes/</releaseNotes>
```

Comment:
@gep13 Hi!  Thanks for the comment!  It is not necessary to break this out into a newline for a clicky link.  Any fully valid URI is automatically converted into a clickable link by markdownsharp, regardless of it's position.  I try to just  leave a static link for releaseNotes tag because there isn't a good way for now to automatically update the content in this section with chocopkgup.  Of course some people ::cough:: @ferventcoder ::cough:: think it better to leave the most epic releaseNotes ever: https://github.com/chocolatey/choco/blob/master/nuget/chocolatey/chocolatey.nuspec#L40-L142